test/integration/connect/envoy/run-tests.sh

```diff
@@ -1,57 +1,20 @@
-#!/bin/bash
+#!/usr/bin/env bash
 
 set -eEuo pipefail
 
 # DEBUG=1 enables set -x for this script so echos every command run
 DEBUG=${DEBUG:-}
 
-# FILTER_TESTS="<pattern>" skips any test whose CASENAME doesn't match the
-# pattern. CASENAME is combination of the name from the case-<name> dir and the
-# envoy version for example: "http, envoy 1.8.0". The pattern is passed to grep
-# over that string.
-FILTER_TESTS=${FILTER_TESTS:-}
-
-# STOP_ON_FAIL exits after a case fails so the workdir state can be viewed and
-# the components interacted with to debug the failure. This is useful when tests
-# only fail when run as part of a whole suite but work in isolation.
-STOP_ON_FAIL=${STOP_ON_FAIL:-}
-
-# ENVOY_VERSIONS is the list of envoy versions to run each test against
-ENVOY_VERSIONS=${ENVOY_VERSIONS:-"1.11.2 1.12.3 1.13.1 1.14.1"}
+# ENVOY_VERSION to run each test against
+ENVOY_VERSION=${ENVOY_VERSION:-"1.14.1"}
+export ENVOY_VERSION
 
 if [ ! -z "$DEBUG" ] ; then
   set -x
 fi
 
-DIR=$(cd -P -- "$(dirname -- "$0")" && pwd -P)
-
-cd $DIR
-
-LEAVE_CONSUL_UP=${LEAVE_CONSUL_UP:-}
-PROXY_LOGS_ON_FAIL=${PROXY_LOGS_ON_FAIL:-}
-
 source helpers.bash
 
-RESULT=1
-CLEANED_UP=0
-
-function cleanup {
-  local STATUS="$?"
-
-  if [ "$CLEANED_UP" != 0 ] ; then
-    return
-  fi
-  CLEANED_UP=1
-
-  if [ "$STATUS" -ne 0 ]
-  then
-    capture_logs
```

Comment:
This `capture_logs` is not called from `suite_teardown`. The `STATUS` var won't have the right value anymore. We could pass an env var to indicate sucess/failure , and capture logs when it is set to failure.

I dropped this call to `capture_logs` because:
1. `capture_logs` is already called from all of the failure conditions in `run_tests`. I think those calls should capture everything
2. `capture_logs` expects a `LOG_DIR` set to a test directory. But `cleanup` is called as "suite teardown". So logs would end up in the directory for the case that happened to run last.

If I have misunderstood this call and this `capture_logs` is intended to do something else I can restore it using the env var.